src/device_trezor/device_trezor_base.cpp

```diff
@@ -151,7 +152,8 @@ namespace trezor {
 
     bool device_trezor_base::disconnect() {
       TREZOR_AUTO_LOCK_DEVICE();
-      m_device_state.clear();
+      m_device_session_id.wipe();
```

Comment:
```suggestion
```

Reply:
You mean wipe is not needed? I need to invalidate the state, so maybe just `.clear()`?

Reply:
> 
> 
> You mean wipe is not needed? I need to invalidate the state, so maybe just `.clear()`?

Right. You are clearing it on the next line, so wipe is not needed.

Reply:
Aah I've just double-checked and clear indeed wipes the buffer (as it should). I've just wrongly assumed it just resets the size. Thanks!